src/tagmanager/tm_tag.h

```diff
@@ -94,14 +101,15 @@ typedef struct TMTag
 	TMSourceFile *file; /**< File in which the tag occurs; NULL for global tags */
 	gulong line; /**< Line number of the tag */
 	gboolean local; /**< Is the tag of local scope */
-	guint pointerOrder;
+	guint flags; /**< Additional flags */
```

Comment:
This adds Doxygen documentation for the member, which traditionally approves it for plugin use. Is that your intention? Otherwise I would add a simple comment (like with kind_letter) until someone officially requests this member to be part of the API.

Reply:
This was intentional - plugins should know if a tag is anonymous (I currently don't do it right in my ProjectOrganizer plugin but in Find Tag where you can find a tag by name I should ignore anonymous tags).

I didn't do this with the kind_letter because there I really don't see much reason why plugins should look at it (and in addition, it may not always be present in the TMTag structure, for instance when the tag is read from our custom tag file format).